src/Model/ApplicationSchema.php

```diff
@@ -187,32 +128,32 @@ public function __construct(array $data = null)
     }
 
     /**
-     * Show all the invalid properties with reasons.
+     * show all the invalid properties with reasons.
      *
      * @return array invalid properties with reasons
      */
     public function listInvalidProperties()
     {
-        $invalidProperties = [];
+        $invalid_properties = [];
 
-        return $invalidProperties;
+        return $invalid_properties;
     }
 
     /**
-     * Validate all the properties in the model
+     * validate all the properties in the model
      * return true if all passed
      *
      * @return bool True if all properties are valid
      */
     public function valid()
     {
-        return count($this->listInvalidProperties()) === 0;
+
+        return true;
```

Comment:
I guess this changed with the new generation. Not sure if it's an immediate thing to address, but I feel like it should be noted.